### PR TITLE
Create dashboardhub.md

### PR DESCRIPTION
**Pull request by eddiejaoude**

### CODE REVIEW

**Comment by skoop** [comment]
IWantTo.Support was made mostly to give an overview of how to do financial donations to open source projects. This PR does not actually list any ways of financially supporting the project. Are financial donations possible?

**Comment by eddiejaoude** [comment]
Hi @skoop, Thanks for your feedback. I did not realise it was just for financial donations. I have now added a paypal [donation button](http://dashboardhub.io/#donate) in the footer of the website. I understand our project is very new and appreciate if you wish to keep this for the more mainstream projects. Thanks

File: dashboardhub.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+slug: dashboardhub
+---
+
+# DashboardHub.io
+
+## What is it?
+
+The initial aim of **DashboardHub** is to give *transparancy* and *visibility* to the Development Pipeline. 
+
+There are serveral sub projects which are microservices for the **DashboardHub** service architecture, from creating a Dashboard to Beaconing YSlow Data.
```
Comment (skoop) [inline dashboardhub.md]: there's a typo here: several

File: dashboardhub.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+More information on the [website](http://dashboardhub.io)
+
+## How to support
+
+Supporting the project can be any of the following:
+
+* Star / Favourite on Github
```
Comment (skoop) [inline dashboardhub.md]: could you rewrite this to talk about the Paypal donation option?

**Comment by skoop** [comment]
It should not just be about mainstream projects, but it _is_ about financial donations. Most people already know about pull requests, documentation, etc. People seem to not be aware that donating some money (whether beer-money or something else) is an option as well. That's why I created this site.

**Comment by eddiejaoude** [comment]
I like your idea! :+1:

**Pull request merged**